src/service/profile.js

```diff
@@ -141,7 +141,7 @@ export const setFollow = async (data) => {
   const textTargetUser = `${data.username_follower} started following you.\nSend them a message now`;
   const textOwnUser = `You started following ${data.username_followed}.\nSend them a message now.`;
   const members = [data.user_id_follower, data.user_id_followed];
-  await followClient(members, data, textTargetUser, textOwnUser);
+  followClient(members, data, textTargetUser, textOwnUser);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of the `await` keyword before the `followClient` function call could potentially lead to race conditions. If `followClient` is asynchronous and if its completion is required before proceeding, it should be awaited. Otherwise, the API call to '/profiles/follow-user-v3' might execute before `followClient` has completed, which could lead to inconsistent data.

```diff
- followClient(members, data, textTargetUser, textOwnUser);
+ await followClient(members, data, textTargetUser, textOwnUser);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->